### PR TITLE
Use text-color icons for system tray Send/Receive menu entries

### LINKED ISSUE #5625: [QT] Win: icon coloring issue tray-menu
@luke-jr would you mind have a look at it?

bildschirmfoto 2015-01-09 um 11 13 47

Light blue color for std. theme is okay.
I still would prefer the black one... but i'm fine with the auto-color.

bildschirmfoto 2015-01-09 um 11 03 46
bildschirmfoto 2015-01-09 um 11 15 47

classic theme also looks okay:
bildschirmfoto 2015-01-09 um 11 16 27

### PR DESCRIPTION
**Pull request by luke-jr**
Fixes #5625

### CODE REVIEW

**Comment by jonasschnelli** [comment]
@zander please have a look at: #5625. I think the problem addresses in the PR is win only.

**Comment by zander** [comment]
I suggest using #ifdef Q_OS_WIN then for this code; so it won't make things worse on other OS's (esp linux) where this behavior change is not wanted.

**Comment by jonasschnelli** [comment]
@zander in general we like to keep the platform dependent stuff (#ifdef) on a minimum. And the problem with mixed colors could occur on other platforms.

**Comment by jonasschnelli** [comment]
Tested on Win,OSX,Ubuntu (change only visible on Windows, see below).

Tested ACK.
But it brought me to a new issue: #5653
bildschirmfoto 2015-01-13 um 08 28 43

**Comment by zander** [comment]
> And the problem with mixed colors could occur on other platforms.

The "problem" is debatable. Having the same icon in all places is quite important for a stable UI (consistency).
Anywhere I look having monochrome icons that follow the text color just isn't the default, and the menu in the latest screenshot looks broken. Its completely inconsistent with all the other apps because they all use color icons. Its also inconsistent with itself because the "Show / Hide" for instance is not actually the logo you see in the taskbar, its got a different color.

Bottom line, I have zero apps on either Windows (8.1) nor on KDE that have monochrome icons in their systray context menu.  So this just doesn't look consistent in any way at all.

**Comment by jonasschnelli** [comment]
@zander The discussion over the monochrome icons has already been held. But feel free to contribute over issue and new pull requests. This pull requests focus on #5625.

**Comment by laanwj** [comment]
ACK.

File: src/qt/bitcoingui.h [hunk 1]
```diff
@@ -90,12 +90,14 @@ class BitcoinGUI : public QMainWindow
     QAction *historyAction;
     QAction *quitAction;
     QAction *sendCoinsAction;
+    QAction *sendCoinsMenuAction;
```
Comment (laanwj) [inline src/qt/bitcoingui.h]: These extra fields neeed to be zeroed in the constructor

Comment (luke-jr) [inline src/qt/bitcoingui.h]: createActions, which is part of the constructor, initialises them. Regardless, I added the (unnecessary) zeroing as requested.

Comment (laanwj) [inline src/qt/bitcoingui.h]: Yes, it's mostly to satisfy static checkers, as well as a belt-and-suspenders measure in general. Thanks.
